src/query-store.ts

```diff
@@ -37,6 +37,11 @@ export interface UseQueryEntry<TResult = unknown, TError = unknown> {
    */
   state: ShallowRef<DataState<TResult, TError>>
 
+  /**
+   * A placeholder data that is initially shown while the query is loading for the first time.
```

Comment:
@posva: should the error case be mentioned?

Reply:
yes but in the options definition, not here